src/cosmosdb-preview/azext_cosmosdb_preview/vendored_sdks/azure_mgmt_cosmosdb/_cosmos_db_management_client.py

```diff
@@ -122,6 +128,7 @@ def __init__(
         super(CosmosDBManagementClient, self).__init__(self.config.credentials, self.config)
 
         client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
+        self.api_version = '2021-03-01-preview'
```

Comment:
Can you try using this command to generate the sdk :
autorest --version=V2 --use=@microsoft.azure/autorest.python@~4.0.71 --python --python-mode=update --multiapi --keep-version-file --no-async --python-sdks-folder=E:\CosmosDBClientSDKs\azure-sdk-for-python\sdk2 ./readme.md

Reply:
I have picked the sdk from this location. sdk generated  by PR
https://github.com/Azure/azure-rest-api-specs/pull/12861/checks?check_run_id=1964851992